Library/Styles/CheckoutStyles.swift

```diff
@@ -109,6 +109,12 @@ public let checkoutStackViewStyle: StackViewStyle = { (stackView: UIStackView) i
     |> \.spacing .~ (Styles.grid(1) + Styles.gridHalf(1))
 }
 
+public let checkoutSwitchControlStyle: SwitchControlStyle = { switchControl in
+  switchControl
+    |> \.onTintColor .~ UIColor.ksr_green_500
+    |> \.tintColor .~ UIColor.ksr_green_500
```

Comment:
Is the `tintColor` supposed to be green? Our app usually adopts grey for the `off` state. Maybe a `ksr_grey_500` or darker would fit better?

Reply:
Yeah, the design was kinda weird because it wasn't using the native switch component...let me run this by Nneka